pages/release-process.md

```diff
@@ -146,68 +146,108 @@ When the vote passes on a release candidate, follow the steps below to complete
-
-1. Run the integration tests of projects that depend on Fluo (using the release candidate jars)
-
-1. Run example Fluo applications (using the release candidate tarball)
-
-These strategies are described below with step-by-step instructions.
-
-#### Run integration tests
-
-A Fluo release can be tested by running the integration tests of projects that use Fluo.
-
-1. Before running integration tests, you'll need the following information:
-    * Release version (example: `1.2.0`)
-    * Staging repo (example: `https://repository.apache.org/content/repositories/orgapachefluo-1023/`)
-        * Note: The staging repo changes with each release candidate.
-
-1. Add the following configuration to `~/.m2/settings.xml` to instruct Maven to build using the staging repo
-   for the release candidate. Also, make sure that the `fluoRC` profile is active while you are testing the release
-   but comment out `<activeProfiles>` section after you are done testing.
-    ```xml
-    <profiles>
-      <profile>
-        <id>fluoRC</id>
-        <repositories>
-          <repository>
-            <id>fluorc</id>
-            <name>fluorc</name>
-            <url>https://repository.apache.org/content/repositories/orgapachefluo-1023/</url>
-          </repository>
-        </repositories>
-        <pluginRepositories>
-          <pluginRepository>
-            <id>fluorcp</id>
-            <name>fluorcp</name>
-            <url>https://repository.apache.org/content/repositories/orgapachefluo-1023/</url>
-          </pluginRepository>
-        </pluginRepositories>
-      </profile>
-    </profiles>
-    <activeProfiles>
-      <activeProfile>fluoRC</activeProfile>
-    </activeProfiles>
-    ```
-1. Clone a project that uses Fluo like the [Fluo Recipes] project below:
+#### Set up your environment
+
+Set up required for all tests:
+
+1. Set the release version, staging repo, and path to release Maven settings:
+   ```shell
+   export RC_VERSION=1.2.0
+   export RC_STAGING=https://repository.apache.org/content/repositories/orgapachefluo-1023/
+   export FLUO_MAVEN=/tmp/fluo-maven.xml
+   ```
+1. Create Maven settings
+   ```shell
+   $ cat <<EOF >$FLUO_MAVEN
+   <settings>
+     <profiles>
+       <profile>
+         <id>fluoRC</id>
+         <repositories>
+           <repository>
+             <id>fluorc</id>
+             <name>fluorc</name>
+             <url>\${env.RC_STAGING}</url>
+           </repository>
+         </repositories>
+         <pluginRepositories>
+           <pluginRepository>
+             <id>fluorcp</id>
+             <name>fluorcp</name>
+             <url>\${env.RC_STAGING}</url>
+           </pluginRepository>
+         </pluginRepositories>
+       </profile>
+     </profiles>
+     <activeProfiles>
+       <activeProfile>fluoRC</activeProfile>
+     </activeProfiles>
+   </settings>
+   EOF
+   ```
+1. If a new release candidate is made, update your staging repo.
```

Comment:
Since these instructions show using `/tmp`, there's no reason to suspect that the previous file will still exist against the new RC. It'd probably be better to just have them start over. That could simplify some of this, because there'd be no reason to put the filename in a variable, or to use `\${env.RC_STAGING}` (could use `$RC_STAGING`) instead... or better yet... put most of the URL in the file, and just the number in the variable.

Reply:
I can do that `RC_STAGING` will just be the number (i.e `1023`)

Reply:
I like the idea of using the staging repo number. I pushed a commit that does this.